make_test_images/src/make_test_images.rs

```diff
@@ -93,6 +93,26 @@ impl Default for Config {
     }
 }
 
+/// Generate a blake3 hash over the image in path using a fixed buffer
+fn blake3_hash(path: &Path) -> Result<String> {
```

Comment:
Just a side note.  I added blake3 support to hash_utils so it will be generally available.

Reply:
I'll call that instead of what I wrote when we get things merged.

Reply:
i think that means we will have a Blake3 implementation in in the SDK. The question is if I should use it by default or make it a public API to to called only when desired.

Reply:
I'l integrate with the new implementation after this PR